Makefile.am

```diff
@@ -13,12 +13,14 @@ AM_LFLAGS   = -o lex.yy.c
 
 AM_CC       = $(PTHREAD_CC)
 AM_CFLAGS   += $(PTHREAD_CFLAGS)
-AM_LDFLAGS  += $(PTHREAD_LIBS)
+AM_LDFLAGS  += $(PTHREAD_LIBS) \
+			   $(zlib_LIBS)
```

Comment:
Shouldn't need to add it here, but it does need to be added to `bin_fstdump_LDADD` in `test/Makefile.am`.

Reply:
Here goes, though I will have to check what consequences it has on platforms other than Linux...